.github/workflows/publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: npm-publish
+on:
+  workflow_dispatch:
```

Comment:
this should be as here: https://github.com/redhat-developer/app-services-sdk-js/blob/8c287ccb1ece9f252f32a850d29ca68c4f720c1e/.github/workflows/release.yml#L3-L5